docker/app
# Bump buildx and fix file finalizer breaking the stdout fix

**- What I did**
Replaced os.File for an interface in `containerd/console` (https://github.com/ulyssessouza/console/commit/f652dc3e99a9f4aa760deb9b4b28edb7c4e5001a) so that can be used by `docker/buildx` (https://github.com/ulyssessouza/buildx/commit/5941345e21ebda43723a475380135fe3741d6b3c) and `docker/app`

**- How I did it**
Fork and override of the repos in `Gopkg.toml`

**- How to verify it**
Check that the broken output fix still working by repeating a `docker app build` exhaustively.

**- Description for the changelog**
Use a file interface from `containerd/console` to pass to buildx's printer

**- A picture of a cute animal (not mandatory)**
![mastigando](https://user-images.githubusercontent.com/373485/70074935-064ef080-15fc-11ea-99ee-1df48393dc13.jpg)
